pkg/proxy/iptables/proxier.go

```diff
@@ -765,6 +777,10 @@ func (proxier *Proxier) execConntrackTool(parameters ...string) error {
 // The only other iptables rules are those that are setup in iptablesInit()
 // assumes proxier.mu is held
 func (proxier *Proxier) syncProxyRules() {
+	if proxier.throttle != nil && !proxier.throttle.TryAccept() {
```

Comment:
@thockin - Only question is if we should block on Accept() or put back in the timer adjustment jiggery?